doc/rules/class_notation/class_attributes_separation.rst

```diff
@@ -14,7 +14,7 @@ Configuration
 Dictionary of ``const|method|property|trait_import|case`` =>
 ``none|one|only_if_meta`` values.
 
-Allowed types: ``list<string>``
+Allowed types: ``array``
```

Comment:
nice change as `['const' => 'one', ...]` is not a `list` but a 2d array  :+1:

Reply:
you are totally right, @SpacePossum 
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8081/commits/6374f60da04324394de9a64ad9f6427204faad42#diff-fc6cf624709e515519afc9f06ace3e41741b4dadd2dd48c5dab15963eb0e149aR17